server/src/language-service/opctagListProvider.ts

```diff
@@ -0,0 +1,42 @@
+import { SepticReferenceProvider } from "../septic";
+
+export class OpcTagListProvider {
+    constructor() {}
+
+    public generateOpcTagList(refProvider: SepticReferenceProvider): string {
+        return "";
+    }
+}
+
+export function generateOpcReport(
+    refProvider: SepticReferenceProvider
+): string {
+    let header: string = "ObjectId;ObjectType;ObjectAttribute;OPCTag;";
+    let entries: string[] = [];
+    let opcObjects = refProvider.getObjectsByType(
+        "SopcTvr",
+        "SopcEvr",
+        "SopcCvr",
+        "SopcDvr",
+        "SopcMvr",
+        "SopcProc"
+    );
+    for (let obj of opcObjects) {
+        let objectName = obj.identifier?.id ?? "unknown";
+        let tagAttributes = obj.attributes.filter((attr) =>
+            attr.key.endsWith("Tag")
+        );
+        for (let tagAttr of tagAttributes) {
+            let value = tagAttr.getAttrValue()?.getValue() ?? "";
+            if (
+                value.trim() === "" ||
+                value === "DUMMY_TAG" ||
+                value === "NotUsed"
+            ) {
+                continue;
+            }
+            entries.push(`${objectName};${obj.type};${tagAttr.key};${value};`);
```

Comment:
I think leaving a delimiter at the end of the line is invalid a it indicates a fifth field.
```suggestion
            entries.push(`${objectName};${obj.type};${tagAttr.key};${value}`);
```

Reply:
One more thing: I don't understand this list that I get when I run the command. Surely it can only create lists from generated .cnfg files?

<img width="428" alt="image" src="https://github.com/equinor/vscode-septic/assets/2356425/a4fdc07c-649f-4ad8-9ccd-c5fb24776dc5">

Reply:
Might as well support scg since it might be cases where you want to get an overview of the well "typical" etc

Reply:
Sure, sounds like a good idea. But then there are two issues:
1. When creating the list for a yaml file, the resulting csv contains the tags from the cnfg.
2. As you can see from the screenshot above, ABBSim.cnfg did not show up even though it was already generated. It shows up now, though, without having restarted the debug session, so not sure what is going on.

Reply:
> 2. As you can see from the screenshot above, ABBSim.cnfg did not show up even though it was already generated. It shows up now, though, without having restarted the debug session, so not sure what is going on.

... and now ABBSim.cnfg doesn't show up in the list yet again. Looks like it shows up if I have it opened in an editor windows, but not otherwise. But TRAAPO.cnfg always shows up.